src/components/Setting/ThemeSettingInputs.tsx

```diff
@@ -0,0 +1,39 @@
+interface ThemeSettingData {
+    darkModeWhenRunning: boolean;
+}
+
+interface ThemeSettingInputsProps {
+    themeSetting: ThemeSettingData;
+    setThemeSetting: (value: ThemeSettingData) => void;
+}
+
+const ThemeSettingInputs = ({ themeSetting, setThemeSetting }: ThemeSettingInputsProps) => {
+    const handleChange = () => {
+        setThemeSetting({
+            ...themeSetting,
+            darkModeWhenRunning: !themeSetting.darkModeWhenRunning,
+        });
+    };
+
+    return (
+        <div className="theme-setting">
+            <h3 className="setting-title capita py-4 font-semibold text-gray-400">
```

Comment:
```suggestion
            <h3 className="setting-title capita py-4 font-semibold text-gray-400">
```
my bad, "capita"  -> "capitalize"